src/components/content/catalog/services/unregister/UnregisterResult.tsx

```diff
@@ -31,12 +33,15 @@ export function UnregisterResult({
         });
     };
 
-    if (useUnregisterRequestState[0]) {
+    if (
```

Comment:
is it necessary to change this if condition ?
It must work just with the old condition.